pkg/util/sys/device.go

```diff
@@ -63,6 +63,8 @@ type LocalDisk struct {
 	Rotational bool `json:"rotational"`
 	// ReadOnly is the boolean whether the device is readonly
 	Readonly bool `json:"readOnly"`
+	// Partitions is a partition slice
+	Partitions []Partition
```

Comment:
If we're going to return the partitions, is there a need for the OwnPartition property anymore?

Reply:
good point, ownPartition is now removed